src/Microsoft.OData.Core/Json/JsonLightInstanceAnnotationWriter.cs

```diff
@@ -1,236 +1,237 @@
-﻿//---------------------------------------------------------------------
-// <copyright file="JsonLightInstanceAnnotationWriter.cs" company="Microsoft">

```

Comment:
>JsonLightInstanceAnnotationWriter [](start = 20, length = 33)

This file appears in the diff as a huge delete and add. Earlier versions show small whitespace edits only. Does this file need to be included?